src/components/UserTable/UserTable.jsx

```diff
@@ -24,71 +26,127 @@ import AUTH_ROLES from '../../utils/AuthConfig';
 
 const { SUPERADMIN_ROLE, ADMIN_ROLE } = AUTH_ROLES.AUTH_ROLES;
 
-const UserTable = ({ isSuperAdmin, users }) => {
+const UserTable = ({
+  isSuperAdmin,
+  users,
+  setAllUsers,
+  setDriverUsers,
+  setAdminUsers,
+  updateDisplay,
+}) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { isOpen: deleteIsOpen, onOpen: deleteOnOpen, onClose: deleteOnClose } = useDisclosure();
+  const [editUserData, setEditUserData] = useState([]);
+  const [deleteUserData, setDeleteUserData] = useState([]);
+
+  const openEdit = data => {
+    setEditUserData(data);
+    onOpen();
+  };
+
+  const openDelete = data => {
+    setDeleteUserData(data);
+    deleteOnOpen();
+  };
   return (
-    <TableContainer border="1px" borderColor="gray.200">
-      <Table variant="striped" colorScheme="gray">
-        <Thead bgColor="rgb(247,250,252)">
-          <Tr className={styles['table-header']}>
-            <Th>
-              <Flex align="center">
-                <Image src={peopleIcon} mr={2} color="black" />
-                <Text color="black">Name</Text>
-              </Flex>
-            </Th>
-            <Th>
-              <Flex align="center">
-                <EmailIcon mr={2} color="black" />
-                <Text color="black">Email Address</Text>
-              </Flex>
-            </Th>
-            <Th>
-              <Flex align="center">
-                <PhoneIcon mr={2} color="black" />
-                <Text color="black">Phone Number</Text>
-              </Flex>
-            </Th>
-            {isSuperAdmin ? (
+    <>
+      <TableContainer border="1px" borderColor="gray.200">
+        <Table variant="striped" colorScheme="gray">
+          <Thead bgColor="rgb(247,250,252)">
+            <Tr className={styles['table-header']}>
+              <Th>
+                <Flex align="center">
+                  <Image src={peopleIcon} mr={2} color="black" />
+                  <Text color="black">Name</Text>
+                </Flex>
+              </Th>
+              <Th>
+                <Flex align="center">
+                  <EmailIcon mr={2} color="black" />
+                  <Text color="black">Email Address</Text>
+                </Flex>
+              </Th>
               <Th>
-                <Flex>
-                  <Image src={cardAccount} mr={2} />
-                  <Text color="black">Role</Text>
+                <Flex align="center">
+                  <PhoneIcon mr={2} color="black" />
+                  <Text color="black">Phone Number</Text>
                 </Flex>
               </Th>
-            ) : null}
-            <Th />
-          </Tr>
-        </Thead>
-        <Tbody className={styles['row-text']}>
-          {users.map(user => (
-            <Tr key={`${user.id}`}>
-              <Td>
-                {user.firstName} {user.lastName}
-              </Td>
-              <Td>{user.email}</Td>
-              <Td>{formatPhone(user.phoneNumber)}</Td>
               {isSuperAdmin ? (
```

Comment:
Use `&&` instead of ternary operator so we can properly conditionally render the column label.